### ISSUE TITLE
When pulling InfiniteContainer the item inside disappears

### ISSUE THREAD

**Issue by plumberg** [codenameone/CodenameOne#3490]
**Describe the bug**
In Android devices, if your `InfiniteContainer` contains only one item, when you pull the container "to refresh" fast enough, the item disappears from the screen for a couple of seconds. 
This is not an issue for simulator and iPhone. Also I wasn't able to see this issue with 2+ items in the container.

**To Reproduce**
1. Create and add `InfiniteContainer` with single listing to your app.
2. Build and run it on Android device
3. Try to pull the container fast, maybe a couple of times. 
4. See how the listing disappears from the screen. 

**Expected behavior**
The listing should remain in container, on the display.

**Smartphone (please complete the following information):**
 - Device: Motorola
 - OS: Android 10 API29 and Android 7

**Comment by DurankGts** [comment]
I think that this is the same issue with the tensile drag that don't disable in the ContainerList some times. Why? I don't know...

https://user-images.githubusercontent.com/16245755/131672427-d152abf6-bf8d-4eba-a950-3ecf55e18fe0.mp4

**Comment by DurankGts** [comment]
this a simple test case

         Form hi = new Form("Container issue", new BorderLayout());
         hi.setUIID("MyForm");
         hi.setSafeArea(true);
         hi.setFormBottomPaddingEditingMode(true);
         hi.setScrollableY(false);

        listChat = new Container(BoxLayout.y());
        listChat.setAlwaysTensile(false);
        listChat.setTensileDragEnabled(false);
        listChat.setTensileLength(0);
        listChat.setScrollableY(true);
        listChat.setScrollVisible(false);
        hi.add(BorderLayout.CENTER, listChat);
        hi.add(BorderLayout.SOUTH, new TextField("", "write here..."));
        hi.show();
        updateContainerList();

    private void updateContainerList(){
        for (int i=0;i<100;i++ ) {
            Label l  = new Label("item: "+i);
            Container item = BorderLayout.center(l);
            listChat.addComponent(i,item);
        }
        listChat.revalidate();
    }

this other video

android:9
alcatel 5024A


https://user-images.githubusercontent.com/16245755/131688934-5c47c7c0-efb7-4513-a2c7-1a28121ee2b2.mp4

**Comment by DurankGts** [comment]
the issue is when you tensile drag the Container list from Bottom to TOP. The tensile drag method just work when the user tensile drag from TOP to Bottom

**Comment by DurankGts** [comment]
this same issue occur in iphone xs ios 14.7.1

https://user-images.githubusercontent.com/16245755/131693528-eacc8a59-4070-4bc3-b08a-c480954de105.mp4

**Comment by plumberg** [comment]
> the issue is when you tensile drag the Container list from Bottom to TOP.

Yes, I believe I had this bug when I dragged list from the Bottom to the Top too, so it's no just if you pull it fast like in my original post.

**Comment by DurankGts** [comment]
@shannah  you can reproduce the same behaivor in android, iphone and simultor. Dragging the list from bottom to TOP many times. Next the item in the list disappears.

**Comment by DurankGts** [comment]
Please @shannah  help me with this issue...I create a test case
